ql/src/java/org/apache/hadoop/hive/ql/exec/tez/monitoring/RenderStrategy.java

```diff
@@ -57,7 +60,8 @@ public void update(DAGStatus status, Map<String, Progress> vertexProgressMap) {
       renderProgress(monitor.progressMonitor(status, vertexProgressMap));
       String report = getReport(vertexProgressMap);
       if (showReport(report)) {
-        renderReport(report);
+        final String time = FORMATTER.format(LocalDateTime.now());
```

Comment:
Can we remove the final here?

Reply:
@dengzhhu653 I don't disagree, but what's your thought? To align with other variables?

Reply:
yes, align with other variables and there seems no neccesarry to declare it as final, as no one would change it after the if branch.

Reply:
@dengzhhu653 Thanks for clarifying that! I removed the final.